image-loader/app/lib/UploadStatusTable.scala

```diff
@@ -0,0 +1,23 @@
+package lib
+
+import com.gu.mediaservice.lib.aws.DynamoDB
+import com.gu.mediaservice.lib.aws.DynamoDB.caseClassToMap
+import model.UploadStatus
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+
+class UploadStatusTable(config: ImageLoaderConfig) extends DynamoDB(config, config.uploadStatusTable) {
```

Comment:
Newer DynamoDB table should use Scanamo, a higher level library, for reading and writing to DynamoDB. Leases is currently the only place that does this, but we'd prefer it to use this if possible.